custom/customize.sh

```diff
@@ -29,6 +29,8 @@ fi
 
 if [ -f custom/css/$APP.css ]; then
   cp custom/css/$APP.css src/css/themeoverrides.css
+  mkdir -p public/src/css/
+  cp custom/css/$APP.css public/src/css/themeoverrides.css
```

Comment:
it would be good if this script can clean up the old css file under `/public` each time it runs, and only copy new ones over if there is one. So old css files won't interference with new runs

Reply:
looking into fixing this, good call